src/components/resource/show.js

```diff
@@ -133,9 +139,23 @@ export default class ResourceShow extends Component {
       }
     ];
 
+    let toasts = processErrors(model);
+
+    // if coming from updating any value on managed title in a managed package
+    // show a success toast
+    if (router.history.action === 'PUSH' &&
+        router.history.location.state &&
+        router.history.location.state.isFreshlySaved) {
```

Comment:
We shouldn't need these kinds of properties in order to send toast notifications. It seems very much like this route is doing the previous routes work for it. I understand that it's not something for this PR because this is how the toast notifications work, but I think we need to seriously revisit the mechanism.